app/controllers/charges_controller.rb

```diff
@@ -194,6 +194,13 @@ def create_square_payment_request(
 
     # Creates a pending PaymentIntent. See webhooks_controller to see what
     # happens when the PaymentIntent is successful.
+    
+    # Check for whether a campaign exits. If a campaign exists, then check whether the campaign has a non-profit associated with it. 
```

Comment:
nit: Mind keeping the line length under 120? (Whatever rubocop is configured to)